plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/KtlintExtension.kt

```diff
@@ -103,6 +105,18 @@ internal constructor(
         set(emptySet())
     }
 
+    /**
+     * Baseline file location.
+     *
+     * Default location is `<projectDir>/ktlintBaseline.xml`.
+     *
+     * @since KtLint `0.41.0`
+     */
+    val baseline: RegularFileProperty = objectFactory.fileProperty()
+        .convention(
+            projectLayout.projectDirectory.file("ktlintBaseline.xml")
```

Comment:
Would it be better to make the convention having this inside of a directory instead of in the project directory?

Reply:
which directory do you have in mind? Something like `$projectDir/ktlint/ktlintBaseline.xml`?

Reply:
Have changed it to `$projectDir/config/ktlint/baseline.xml`